src/librustdoc/html/static/main.js

```diff
@@ -374,7 +375,7 @@
             var valLower = query.query.toLowerCase(),
                 val = valLower,
                 typeFilter = itemTypeFromName(query.type),
-                results = {},
+                results = {}, results_in_args = {}, results_returned = {},
```

Comment:
Not really an issue with this PR, but I don't like how there's inconsistent `var` styles being used with some functions having one `var` per line, and others using commas like they're going out of style.

Reply:
I grouped them by "theme". But if you want I can put each on a line.